src/Agent.Worker/Release/ReleaseJobExtension.cs

```diff
@@ -261,19 +254,8 @@ private void CleanUpArtifactsFolder(IExecutionContext executionContext, string a
                 {
                     executionContext.Output(StringUtil.Loc("RMCleaningArtifactsDirectory", artifactsWorkingFolder));
 
-                    try
-                    {
-                        if (Directory.Exists(artifactsWorkingFolder))
```

Comment:
will removing delete directory here impact download of custom task based artifacts?